modules/cluster-logging-collector-legacy-syslog.adoc

```diff
@@ -5,7 +5,7 @@
 [id="cluster-logging-collector-legacy-syslog_{context}"]
 = Forwarding logs using the legacy syslog method
 
-You can use the *syslog* RFC3164 protocol to send logs to destinations outside of your {product-title} cluster instead of the default Elasticsearch log store by creating a configuration file and config map. You are responsible for configuring the external log aggregator to receive the logs from {product-title}.
+You can use the *syslog* RFC3164 protocol to send logs to destinations outside of your {product-title} cluster store by creating a configuration file and config map. You are responsible for configuring the external log aggregator to receive the logs from {product-title}.
```

Comment:
This says "cluster store". Should "store" have been deleted as well, like it was in the above file?

Reply:
Nice catch - my bad.